CharacterCreation/01_00_Species_Overview.txt

```diff
@@ -38,4 +38,20 @@ Species Overview
 
 ===== Tundari =====
 
+    Tundari are fur-coverd bipedal organic mammals. They have whiskers and a 
+tail, each of varying length. Tundari come from the vibrantly alive planet of
+Tundaar. Tundari as a species are fierce, cunning, curious and playful. 
+
+    Tundari can dominate as any class. For example, you could play as the silent
+padfooted assassin, whos enemies never knew she was there until it was too late.
+You could be the big teddy bear of a beserker who is quick to trust and quicker
+to tear apart any foe who would put their party in peril. You could be the 
+curious combat medic who sometimes gets preocuppied with the science behind the
+sights that the galaxy has to show. 
+
+    Play a Tundari if you want to give yourself a bit more of a challence either
+tactically or in roleplay. Tactically, Tundari have slightly more 
+maneuverability and slightly less survivability. In roleplay, it can be very fun
+to play something other than a human. You play a human every day, after all!
```

Comment:
We want to specify at the top of the paragraph that we're comparing Tundari to humans here, or rework some sentences to take out the hanging "more"s.
Suggestion: "if you want to give yourself a tactical or roleplay challenge." or "give yourself a challenge tactically or in roleplay."
 -"and slightly less survivability than humans."

Reply:
I was actually trying to compare Tundari to the average of all of the species, which does basically mean that I'm comparing them to humans... 
What do you think about that?